.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"image": "ghcr.io/rocker-org/devcontainer/r-ver:4",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {},
+		"ghcr.io/rocker-org/devcontainer-features/apt-packages:1": {
+			"packages": "cmake,python3-venv"
+		},
+		"ghcr.io/eitsupi/devcontainer-features/go-task:1": {},
+		"ghcr.io/rocker-org/devcontainer-features/quarto-cli:1": {}
+	},
+	"mounts": [
+		{
+			"source": "devcontainer-pak-cache",
+			"target": "/home/rstudio/.cache/R/pkgcache/pkg",
+			"type": "volume"
+		},
+		{
+			"source": "devcontainer-cargo-cache-${devcontainerId}",
+			"target": "/usr/local/cargo",
+			"type": "volume"
+		}
+	],
+	"updateContentCommand": {
+		"prqlr-build-deps": "task setup-dev"
```

Comment:
Is it a mistake to have prqlr here?

Reply:
Yeah, it looks like this was copied from prqlr so I forgot to change it.
Thanks for pointing this out.